c/validation/adbc_validation.h

```diff
@@ -274,8 +275,8 @@ class StatementTest {
   template <typename CType>
   void TestSqlIngestNumericType(ArrowType type);
 
-  template <enum ArrowTimeUnit TU>
-  void TestSqlIngestTemporalType(const char* timezone);
+  template <enum ArrowType T>
```

Comment:
Originally I tried to provide ArrowType and ArrowTimeUnit as template parameters, but that kept yielding errors like

```sh
/home/willayd/clones/arrow-adbc/c/validation/adbc_validation.cc:1173:114: error: macro "ASSERT_NO_FATAL_FAILURE" passed 2 arguments, but takes just 1
 1173 |   ASSERT_NO_FATAL_FAILURE(TestSqlIngestTemporalType<NANOARROW_TYPE_DURATION, NANOARROW_TIME_UNIT_SECOND>(nullptr));
      |                                                                                                                  ^
In file included from /home/willayd/clones/arrow-adbc/c/validation/adbc_validation.h:26,
                 from /home/willayd/clones/arrow-adbc/c/validation/adbc_validation.cc:18:
/usr/local/include/gtest/gtest.h:2216: note: macro "ASSERT_NO_FATAL_FAILURE" defined here
 2216 | #define ASSERT_NO_FATAL_FAILURE(statement) \
```

I'm not sure if that is a bug with gtest or my lack of C++ template knowledge, but figured changing the template around like this wasn't a huge deal

Reply:
Oh for future reference, I think you just need `MACRO((Template<A, B>), ...)`